package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "metal-state-validators",
+  "version": "1.0.0",
+  "description": "Utility class with helpers for creating validators for Metal.js state",
+  "main": "lib/type.js",
+  "scripts": {
+    "compile": "babel --presets es2015 -d lib/ src/",
+    "prepublish": "npm run compile",
+    "test": "mocha --compilers js:babel-core/register ./test/**/*.js"
```

Comment:
I initially started with gulp-metal, but then I thought it might not make sense because there are no metal components being used or anything metal related being imported.

If that isn't an issue, I will add it back and use gulp for all of the tasks.